### PR TITLE
Change domain to discord.com

### PR DESCRIPTION
**Pull request by Phineas**
Discord recently announced that they've switched their service from discordapp.com to discord.com and require all developers to comply by November 7th, 2020

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: docs/examples/webhook.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const Discord = require('discord.js');
 /*
  * Create a new webhook
  * The Webbooks ID and token can be found in the URL, when you request that URL, or in the response body.
- * https://discordapp.com/api/webhooks/12345678910/T0kEn0fw3Bh00K
+ * https://discord.com/api/webhooks/12345678910/T0kEn0fw3Bh00K
  *                                     ^^^^^^^^^^  ^^^^^^^^^^^^ 
  *                                     Webhook ID  Webhook Token
```
Comment (tipakA) [inline docs/examples/webhook.js:14]: Very tiny nit: those could be realigned to keep displaying correctly in examples.

Comment (Phineas) [reply docs/examples/webhook.js]: 👍 Resolved

File: src/util/Constants.js [hunk 2]
```diff
@@ -74,13 +74,13 @@ exports.DefaultOptions = {
...
-   * @property {string} [cdn='https://cdn.discordapp.com'] Base url of the CDN
+   * @property {string} [api='https://discord.com/api'] Base url of the API
+   * @property {string} [cdn='https://cdn.discord.com'] Base url of the CDN
    * @property {string} [invite='https://discord.gg'] Base url of invites
    */
   http: {
     version: 7,
-    api: 'https://discordapp.com/api',
+    api: 'https://discord.com/api',
     cdn: 'https://cdn.discordapp.com',
```
Comment (ItsPapaia) [inline src/util/Constants.js:84]:
```suggestion
    cdn: 'https://cdn.discord.com',
```

Comment (vladfrangu) [reply src/util/Constants.js:84]: The CDN doesn't move, and if I saw the conversation in DDevs right, it may not move. Remains to be seen

**Comment by Phineas** [comment]
The CDN has not updates and a Discord developer (night) has said that they likely won’t move the CDN to the new domain due to whitelisting issues.

File: src/util/Constants.js (hunk 2 above)
Comment (Phineas) [reply src/util/Constants.js:84]: CDN isn’t moving, see my comment.

**Comment by JaielSoft** [comment]
So does this mean I now HAVE to update to the newest version of Discord? I have been using v11 for a long time now and this is a big change considering all the breaking changes between v11 and v12. Will there be a v11 update maybe to adress the new URL change for the API? I feel this should be an easy thing to do and should be done for all version else all the npm modules will be unusable except the newest version one and should be removed

**Comment by anandre** [comment]
v11 will not receive updates for this change or intents.

**Comment by Papaia** [comment]
I am quoting Souji from a [PR comment](https://github.com/discordjs/discord.js/pull/4639#issuecomment-659937950) earlier:
> **Version 11** will no longer receive **bug fixes** or new **features** going forward. Please update to version 12 at earliest convenience. We expect it to **break near the end of 2020**. The domain change is one of the consequences for this.
> 
> We have an extensive guide on updating from version 11 to version 12 at [discordjs.guide](https://discordjs.guide/additional-info/changes-in-v12.html?v=12)
> 
> **Edit:** This includes that we no longer accept pull requests to version 11 branches.

**Comment by elsemieni** [comment]
Any plans for including this PR as a release, so we can gather it from npm? :)

**Comment by Monbrey** [comment]
I'm sure a discord.js release will be made well before November 7th, yes

**Pull request merged**